classes/form/username_form.php

```diff
@@ -30,9 +32,9 @@ public function definition() {
         }
 
         $user_attributes = array('placeholder'   => util::lang_string('form_username_placeholder'),
-                                 'additionalcss' => util::lang_string('form_username_button_class'),
-                                 'autofocus'     => '',
-                                 'value'         => $username);
+            'additionalcss' => util::lang_string('form_username_button_class'),
```

Comment:
This was previously indented for readability btw.

Reply:
Yea', this was something I wans't even clear why I changed, and I forgot it like that. Corrected it now. thx!